app/src/main/java/com/andela/art/utils/NetworkUtil.java

```diff
@@ -0,0 +1,27 @@
+package com.andela.art.utils;
+
+import android.content.Context;
+import android.net.ConnectivityManager;
+import android.net.NetworkInfo;
+import android.widget.Toast;
+import com.andela.art.R;
+
+/**
+ * Created by Mugiwara_Munyi on 16/03/2018.
+ */
+
+public class NetworkUtil {
+
+  public void isNetworkAvailable(Context context) {
```

Comment:
In addition to notifying a user, what if this method returned a boolean? This way, we can call this method from other sections of our code to check connectivity and decide on how to proceed depending on it's `true` of `false` return value.

Reply:
Done.